common/constants/api.js

```diff
@@ -52,3 +52,7 @@ export const updateUser = userInfo => {
     ...formatUserData(userInfo),
   }).then(({ data }) => data);
 };
+
+/* PUT REQUESTS */
+export const updateResourceVoteCount = ({ id, voteDirection }) =>
+  put(`api/v1/resources/${id}/${voteDirection}`);
```

Comment:
Interesting that this isn't simply a POST. Any explanation?

Reply:
I am no expert in REST architecture but I guess `PUT` is more suitable since we are modifying a resource? But more importantly, this endpoint is configured to accept only `PUT` method AFAIK. I could be wrong so Aaron can verify this. Using other types of methods would yield `405` response.

Reply:
That makes sense to me. In my mind we'd be modifying a table of votes corresponding to a record, but if we're actually modifying a column in a record on the table for resources, PUT is totally the right call.

Reply:
Yeah, the idea is that it's modifying the vote count on the resource. In the actual data model, a vote is its own thing because you need to distinguish whether a certain user has already voted. But the API doesn't reveal who has voted for what, it only reveals the number of votes. So we went with PUT because as far as the API is concerned, you're modifying a resource, not creating/deleting a vote

Reply:
It comes along for the ride via a client-side set authorization head: `Authorization: `Bearer ${token}``.

![Screen Shot 2020-10-09 at 1 31 47 PM](https://user-images.githubusercontent.com/9523719/95628940-e0705480-0a33-11eb-80ac-5a49a55e870a.png)